### PR TITLE
Fix sorting

### PR DESCRIPTION
**Pull request by JinweiZhu**
I fixed the profile page UI display inconsistency locally when clicking on the side nav bar, and I fix the sorting as well.

### CODE REVIEW

File: voluncheer/templates/profiles/posttogallery.html [hunk 1]
```diff
@@ -27,15 +27,11 @@
...
-        <div class="url">
-            <a class="not-implemented-yet" href="#">SETTINGS</a>
-            <hr align="center">
-        </div>
         <div class="url">
             <a href="{% url 'saved_events' %}" >MY VOLUNCHEERS</a>
             <hr align="center">
         </div>
-        {% if user.pk is not curr_user.pk %}
+        {% if user.pk is not volunteer.pk %}
```
Comment (EdenWuyifan) [inline voluncheer/templates/profiles/posttogallery.html]: Why are you changing curr_user to volunteer? I think curr_user is for checking if one is viewing their own profile. No need to differentiate between volunteer and organization.

**Review by EdenWuyifan — CHANGES_REQUESTED** [review]

File: voluncheer/templates/profiles/profile.html [hunk 2]
```diff
@@ -287,27 +283,30 @@ <h2>PAST OPPORTUNITIES</h2>
...
-            <div class="url">
-                <a href="#" onclick="toggleChatVol()">CHAT</a>
-                <hr align="center">
-            </div>
+                <div class="url">
+                    <a href="#" onclick="toggleChatVol()">CHAT</a>
+                    <hr align="center">
+                </div>
             {% endif %}
+            {% if user.pk is curr_user.pk %}
```
Comment (EdenWuyifan) [inline voluncheer/templates/profiles/profile.html:304]: Create Post is for all users. Every user can post on others gallery. This also goes to viewing gallery

File: voluncheer/templates/profiles/profile.html [hunk 3]
```diff
@@ -287,27 +283,30 @@ <h2>PAST OPPORTUNITIES</h2>
         </div>
 
         <div class="sidenav-url">
+            {% if user.pk is curr_user.pk %}
```
Comment (EdenWuyifan) [inline voluncheer/templates/profiles/profile.html:286]: PROFILE shouldn't be restrained from others

**Review by EdenWuyifan — CHANGES_REQUESTED** [review]

File: voluncheer/templates/profiles/profile.html (hunk 3 above)
Comment (JinweiZhu) [reply voluncheer/templates/profiles/profile.html:286]: If it's a different user, then they wouldn't be able to click other user's profile right? Right now, it redirects to the user's own profile when they click on other's profile.

File: voluncheer/templates/profiles/profile.html (hunk 2 above)
Comment (JinweiZhu) [reply voluncheer/templates/profiles/profile.html:304]: Wait, but this just doesn't make sense to me.... How come others can determine the content in my gallery but now myself?

**Review by EdenWuyifan — APPROVED** [review]

**Pull request merged**